.travis/tests.sh

```diff
@@ -113,10 +112,7 @@ elif [ "${1}" == "tck-tomee" ]; then
 
 elif [ "${1}" == "tck-liberty" ]; then
 
-  curl -L -s -o wlp.zip "${LIBERTY_URL}"
-  unzip wlp.zip
-  cp .travis/wlp-server-template.xml wlp/templates/servers/defaultServer/server.xml
-  LIBERTY_HOME="$( cd ./wlp/ && pwd )"
+  source .travis/install-liberty.sh
```

Comment:
Just a side note: At some point it would be great if we could improve and unify all the scripts that are used to start containers and run the various test suites. Not sure yet how to do that. I just wanted to note, that `tests.sh` is getting huge and difficult to maintain.